### ISSUE TITLE
A11y_VSUserStory935073-ContainerToolsWindow_ContainersTab_Keyboard: Tooltip for the "Remove","Start","Stop" and "Show All" buttons are not accessible by keyboard.

### ISSUE THREAD

**Issue by gomathip02** [dotnet/wpf#2410]
Copied from VSO ID: 1028932

Bug Description:
“Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.” 

Environment Details:​
Application name-VS User Story 935073-Container Tools Window
Microsoft Visual Studio Enterprise 2019 Int Preview
Version 16.5.0 Preview 1.0 [29505.209.master]
OS: Win10

Repro Steps:​
1.Install and launch the Visual Studio2019 INT preview.
2.Create a project  after checking the "Enable Docker Support" checkbox under "Advanced label".​
3.Debug the solution using F5.​
4. Navigate to View tab->other windows->Containers.​
5.Use the tab key and try navigating to the toolbar below the "Containers" tab.
6.Observe if the Tooltip for the "Remove", "Start", "Stop" and "ShowAll" buttons are not accessible by keyboard.

Actual:
Tooltip for the "Remove", "Start", "Stop" and "Show All" buttons are not accessible by keyboard.

Expected:

Tooltip for the "Remove", "Start", " Stop" and "Show All" buttons should be accessible by keyboard. As there is no label information associated for the control,so this information should be available to keyboard only users.

Same issue observed for the  below scenario:
1.Logs tab:
1.Install and launch the Visual Studio2019 INT preview.
2.Create a project  after checking the "Enable Docker Support" checkbox under "Advanced label".
3.Debug the solution using F5.
4. Navigate to View tab->other windows->Containers.
5.Navigate to the Logs tab.
6.Observe  tooltip for the "Refresh ,Clear and Stream log" buttons are not accessible by keyboard.

2.Files tab:
1.Install and launch the Visual Studio2019 INT preview.
2.Create a project  after checking the "Enable Docker Support" checkbox under "Advanced label".
3.Debug the solution using F5.
4. Navigate to View tab->other windows->Containers.
5. Navigate to the Files tab.
6. .Observe  tooltip for the "Refresh" buttons are not accessible by keyboard.

User Impact:
Keyboard only user finds it difficult to traverse through the screen.

Recommendations:​​
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx
​
MAS Reference:
MAS 2.1.1-
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}
MAS1.1.1-
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2d2051f-bdc8-4af7-8e18-38aae867e216}

**Comment by anjalisheel-wpf** [comment]
@gomathip02 
I investigated this bug and I analyzed that this bug is specific to visual studio team and not wpf team as we do not own the code for the "containers" window. So please assign it to the correct team.

**Comment by anjalisheel-wpf** [comment]
@gomathip02 
Please raise this issue in the visual studio forum, find link below :
https://developercommunity.visualstudio.com/search?space=8
